docs/AA--Javascript-in-EE/data/eventespresso/core/selectors/model-specific.md

```diff
@@ -0,0 +1,12 @@
+The following selectors for model specific queries that are available on the object returned by `wp.data.select( 'eventespresso/core' )`.
+
+## `getLatestCheckin( registrationId, dateTimeId )`
+
+This retrieves the latest checkin entity from the state for the given value.  There is a corresponding resolver so if this selector has not been resolved yet for the given arguments, the resolver will retrieve the latest checkin record from the server and assign it to the state as an instance of `BaseEntity`.
```

Comment:
> for the given values.

plural since more than one value is supplied, or maybe:

> for the given arguments. 

since that is the phrasing used in the sentence that follows the above